### PR TITLE
A few minor fixes

### PR DESCRIPTION
**Pull request by cassella**
A few minor fixes in the spec:
* chapel_listing: alphabetize export <=> extern
* Lexical_structure:
  * Add keywords from chapel_listing.tex.
  * 0xreal.frac fractional part is hex digits
* Statements: reorder param-for-statement: and its constituent production
* Types: type aliases: reference type-specifier, not type-part

and one cleanup:
* readWriteComplexTuple.stdin: remove .stdin file of deleted test

### CODE REVIEW

File: test/trivial/shannon/readWriteComplexTuple.stdin [hunk 1]
```diff
@@ -1,2 +0,0 @@
-(9.8, 7.6)
```
Comment (ben-albrecht) [inline test/trivial/shannon/readWriteComplexTuple.stdin]: Was this intentionally included?

Comment (cassella) [inline test/trivial/shannon/readWriteComplexTuple.stdin]: Yes.  It was the first thing I put on my misc branch with the idea of collecting a few other small unrelated things.  It's just that the next five unrelated things turned out to be related.

I could restructure it if you'd prefer.

**Comment by vasslitvinov** [comment]
@cassella Please provide the propose PR message. It goes on the top of this tab, where it currently says "No description provided."

**Comment by vasslitvinov** [comment]
"proposed", that is

**Comment by cassella** [comment]
@vasslitvinov Done, sorry.

**Comment by vasslitvinov** [comment]
This is good to merge. Thank you Paul!

**Pull request merged**